### ISSUE TITLE
Docs for new classifier 

### ISSUE THREAD

**Issue by elyase** [rhiever/sklearn-benchmarks#43]
What would be the easiest way / best starting point to test a new classifier using the same testing procedure/datasets as in the paper?

**Comment by rhiever** [comment]
Yes, unless you want to rerun everything. Might not be a bad idea to to rerun everything just to be thorough.

**Comment by elyase** [comment]
@rhiever I might rerun everything, can you give me a rough idea on how much computing it would need?
But my original question is, if I want to test a new classifier, where should I start?

**Comment by rhiever** [comment]
All of the parameter tuning code is in these directories: https://github.com/rhiever/sklearn-benchmarks/tree/master/model_code

and the code to analyze the results of those searches is in this notebook: https://github.com/rhiever/sklearn-benchmarks/blob/master/notebooks/analyze-sklearn-benchmark6.ipynb
